### ISSUE TITLE
URGENT: After 6.2 upgrade cannot install (removed) Antivirus and Mailfilter

### ISSUE THREAD

**Issue by sipi58** [zentyal/zentyal#1973]
I upgraded my Zentyal 6.1 to 6.2 last day. I fully updated 6.1 before upgraded it. After upgrade not installed (removed) Antivirus and Mailfilter modules. I tried to install on terminal:

sudo apt-get install zentyal-antivirus

The following packages have unfulfilled dependencies:
Depends on: libclamav9 (= 0.102.2+dfsg-0ubuntu0.18.04.1) de csak 0.102.3+dfsg-0ubuntu0.18.04.1 install
Depends on: clamav (= 0.102.2+dfsg-0ubuntu0.18.04.1)
Depends on: clamav-base (= 0.102.2+dfsg-0ubuntu0.18.04.1) de csak 0.102.3+dfsg-0ubuntu0.18.04.1 install
Depends on: clamav-daemon (= 0.102.2+dfsg-0ubuntu0.18.04.1) but it is not marked for installation
Depends on: clamav-freshclam (= 0.102.2+dfsg-0ubuntu0.18.04.1) de csak 0.102.3+dfsg-0ubuntu0.18.04.1 install

You can see the problem is dependency: removed 0.102.2+... packages from repository and I think of sign (=) not correct may be it will be equal or bigger... if it works..

sudo apt-get install zentyal-mailfilter

The following packages have unfulfilled dependencies:
zentyal-mailfilter : Depends on: zentyal-antivirus but it is not marked for installation

**Zentyal OS (please complete the following information):**
 - Version: 6.2
 - Version of the module zentyal-antivirus and zentyal-mailfilter

**Other OS (please complete the following information):**
 - OS: Ubuntu 18.04.04 64 bit

Urgent help is needed.

Tanks and Regards

**Comment by nahuuss** [comment]
Quick & Dirty solution for me:

may not be required, but ...
backup of /var/lib/apt/lists/packages.zentyal.org_zentyal_dists_6.2_main_binary-amd64_Packages
backup of /var/lib/apt/lists/packages.zentyal.org_zentyal_dists_6.2_main_binary-i386_Packages

Update in both files the section "Package: zentyal-antivirus" by replacing all "(= 0.102.2+dfsg-0ubuntu0.18.04.1)" with "(>= 0.102.2+dfsg-0ubuntu0.18.04.1)"

backup of /var/lib/dpkg/status

Update also the section "Package: zentyal-antivirus" by replacing all "(= 0.102.2+dfsg-0ubuntu0.18.04.1)" with "(>= 0.102.2+dfsg-0ubuntu0.18.04.1)"

Do not run an apt-get clean/autoclean or whatever here.

After a reboot I could install AV and Mail Filter without any issues.


From here: https://forum.zentyal.org/index.php/topic,34976.msg113591.html#msg113591

**Comment by ivosm** [comment]
This is Duplicate of #1970

**Comment by sipi58** [comment]
Problem isn't fixed.

I made what you recommend and I can install Antivirus and Mailfilter module but I can't start Antivirus...

sudo zs antivirus status
Zentyal: status module antivirus:                       [ STOPPED ]

I try to start:

sudo zs antivirus start
 * Restarting Zentyal module: antivirus                                                                                                [ OK ] 

But antivirus not run!
sudo zs antivirus status
Zentyal: status module antivirus:                       [ STOPPED ]

I found only this in zentyal.log:
2020/06/03 23:51:06 INFO> Service.pm:965 EBox::Module::Service::restartService - Restarting service for module: antivirus

**Comment by sipi58** [comment]
The # 1970 issue (Antivirus module 6.1.3 error)  is not fixed because the antivirus update can be installed but cannot be started. I don't know that this is a new mistake but it is certainly related.

Originally, I also installed the Antivirus module 6.1.3 update on Zentyal 6.1 but the module could not be started. That's why I also upgraded to the Zentyal 6.2 but the Antivirus module cannot be started even after the proposed manual changes... One is worse than the other.

Without Antivirus the system is not secure.

**Comment by brunolorente** [comment]
Hi @sipi58, the issue is already fixed, check this out https://github.com/zentyal/zentyal/issues/1970

many thanks for your time!

**Comment by sipi58** [comment]
Hi brunolorente!

I'm saying exactly that it's not fixed.
The error still persists you cannot start the Antivirus module!

**Comment by sipi58** [comment]
Hi @brunolorente!

I installed new update Antivirus 6.2.1 but the module will not start after the update. I restarted the server but Antivirus will not start after restarting. And I can't start it manually...

sudo zs antivirus status
Zentyal: status module antivirus: [ STOPPED ]

I try to start:

sudo zs antivirus start
Restarting Zentyal module: antivirus [ OK ]

But antivirus not run!

sudo zs antivirus status
Zentyal: status module antivirus: [ STOPPED ]
